src/ui_basic/listselect.h

```diff
@@ -278,6 +351,23 @@ template <typename Entry> struct Listselect<Entry&> : public Listselect<Entry*>
 	Entry& get_selected() const {
 		return *Base::get_selected();
 	}
+
+	bool is_checked(Entry& entry) const {
+		return Base::is_checked(&entry);
+	}
+
+	bool set_checked(Entry& entry, bool newstate, bool notify = false) {
+		return Base::set_checked(&entry, newstate, notify);
+	}
+
+	bool toggle_checked(Entry& entry, bool notify = false) {
+		return Base::toggle_checked(&entry, notify);
+	}
+
+protected:
+	void emit_checkmark_changed(uint32_t base_entry, bool newstate) const override {
+		checkmark_changed(*Base::entry_cache_[base_entry], newstate);
+	}
```

Comment:
#6534 removes support for `Listselect<Entry&>` so everything after L310 can be culled when that goes through.